src/test/ts/atomic/UtilsTest.ts

```diff
@@ -1,15 +1,30 @@
 import { Assertions } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
+import { describe, it } from '@ephox/bedrock-client';
 import { isValidKey } from 'src/main/ts/Utils';
 
-UnitTest.test('UtilsTest', () => {
-
+describe('UtilsTest', () => {
   const checkValidKey = (key: string, expected: boolean) => {
     const actual = isValidKey(key);
-    Assertions.assertEq('Key is valid', expected, actual);
+    Assertions.assertEq('Key should be valid in both camelCase and lowercase', expected, actual);
   };
 
-  checkValidKey('onKeyUp', true);
-  checkValidKey('onkeyup', true);
-  checkValidKey('onDisable', false);
+  // eslint-disable-next-line max-len
+  // v-on event listeners inside DOM templates will be automatically transformed to lowercase (due to HTML’s case-insensitivity), so v-on:myEvent would become v-on:myevent. ref: https://eslint.vuejs.org/rules/custom-event-name-casing
+
+  describe('Valid event name tests', () => {
+    const validKeys = [
+      { key: 'onKeyUp', description: 'camelCase event name "onKeyUp"' },
+      { key: 'onkeyup', description: 'lowercase event name "onkeyup"' }
+    ];
+
+    validKeys.forEach(({ key, description }) => {
```

Comment:
```suggestion
    Arr.each(validKeys, ({ key, description }) => {
```

Reply:
Update in [this](https://github.com/tinymce/tinymce-vue/pull/434/commits/d5b7244dcb95121f0af5989c16c8fb6d2cecf822) commit and retested to verify.